### PR TITLE
NAS-102963 stable

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

**Comment by themylogin** [comment]
Unfortunately that all is outdated. Let me show you how current B2 UI looks like:



They've simplified everything, now it's "keyID" and "applicationKey" and master key is unified with other ones.

Please update title/tooltip according to screenshot above. Hope they won't change it again soon.

**Comment by lola27** [comment]
@wblock I removed the tooltips since the advice how to find the key is incorrect after B2 changed. could doc team regenerate the tooltips? Thanks.

File: src/app/helptext/system/cloudcredentials.ts [hunk 1]
```diff
@@ -87,34 +87,13 @@ export const helptext_system_cloudcredentials = {
...
- or applicationKeyID by logging in to the account,\
- clicking <i>Buckets</i>, and clicking\
- <i>Show Account ID and Application Key</i>. Enter the\
- <i>Account ID</i> to associate the entire account or\
- generate a new <i>Application Key</i>. The <i>keyID</i>\
- replaces the Account ID and the key string is used in\
- place of the <i>Master Application Key</i>.'
-    )
+    placeholder: T("Key ID"),
+    tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/system/cloudcredentials.ts]: Alphanumeric `<a href="https://www.backblaze.com/b2/cloud-storage.html" target="_blank">Backblaze B2</a>` Application Key ID. To generate a new application key, log in to the Backblaze account, go to the `<i>App Keys</i>` page, and add a new application key. Copy the application `<i>keyID</i>` string to this field.

File: src/app/helptext/system/cloudcredentials.ts [hunk 2]
```diff
@@ -87,34 +87,13 @@ export const helptext_system_cloudcredentials = {
...
- page. <a\
- href="https://help.backblaze.com/hc/en-us/articles/224991568-Where-can-I-find-my-Account-ID-and-Application-Key-"\
- target="_blank">Generating a new Master Application Key</a>\
- will invalidate the existing Master key and require\
- updating this field. Using a limited permissions\
- Application Key also requires changing the\
- <i>Account ID</i> to the new <i>keyID</i>.'
-    )
+    placeholder: T("Application Key"),
+    tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/system/cloudcredentials.ts]: `<a href="https://www.backblaze.com/b2/cloud-storage.html" target="_blank">Backblaze B2</a>` Application Key. To generate a new application key, log in to the Backblaze account, go to the `<i>App Keys</i>` page, and add a new application key. Copy the `<i>applicationKey</i>` string to this field.

File: src/app/helptext/system/cloudcredentials.ts (hunk 1 above)
Comment (wblock) [inline src/app/helptext/system/cloudcredentials.ts]:
```suggestion
    tooltip: T('Alphanumeric
<a href="https://www.backblaze.com/b2/cloud-storage.html" \
target="_blank">Backblaze B2</a> Application Key ID. To \
generate a new application key, log in to the Backblaze account, \
go to the <i>App Keys</i> page, and add a new application key. \
Copy the application <i>keyID</i> string to this field.'),
```

File: src/app/helptext/system/cloudcredentials.ts (hunk 2 above)
Comment (wblock) [inline src/app/helptext/system/cloudcredentials.ts]:
```suggestion
    tooltip: T('<a href="https://www.backblaze.com/b2/cloud-storage.html" \
target="_blank">Backblaze B2</a> Application Key. To generate \
a new application key, log in to the Backblaze account, go to the \
<i>App Keys</i> page, and add a new application key. Copy the \
<i>applicationKey</i> string to this field.'),
```

**Pull request merged**